vowels-only.js

```diff
@@ -0,0 +1,57 @@
+// // Write a function called vowelsOnly which accepts a string str 
+// // and returns a new string consisting of just the vowels in str.
+
+// function vowelsOnly(str) {
+
+//     var matches = str.match(/[aeiou]/gi);
+//     // console.log("matches", matches);
+
+//     var newStr = "";
+
+//     if (!matches) {
+//         return "";
+//     }
+
+//     for (var i = 0; i < matches.length; i++) {
+//         newStr = newStr + matches[i];
+//     }
+
+//     return newStr;
+// }
+// console.log(vowelsOnly("hello world")); // "eoo"
+// console.log(vowelsOnly("happy birthday to you")); // "aiaoou"
+// console.log(vowelsOnly("SHOUT it out")); // "OUiou"
+// console.log(vowelsOnly("aieeeeeeeeou")); // "aieeeeeeeeou"
+// console.log(vowelsOnly("nvwlsnlycnsnnts")); // ""
+
+
+function countVowels(str) {
```

Comment:
Since this is in a different file, remove from here. and uncomment the "vowelsOnly"